3.5/sdk/windowsservercore-1803/Dockerfile

```diff
@@ -45,8 +45,7 @@ RUN `
 RUN curl -fSLo sdk_tools48.zip https://dotnetbinaries.blob.core.windows.net/dockerassets/sdk_tools48.zip `
     && tar -zxf sdk_tools48.zip `
```

Comment:
Thoughts on utilizing a temp directory to extract the contents to and then simply removing the directory?  It feels like that is a safer more robust pattern.